src/ipc/mem/FlexibleArray.h

```diff
@@ -27,20 +27,15 @@ class FlexibleArray
 {
 public:
     explicit FlexibleArray(const int capacity) {
-        if (capacity > 1) // the first item is initialized automatically
-            new (raw()+1) Item[capacity-1];
+        new (raw()) Item[capacity];
     }
 
-    Item &operator [](const int idx) { return items[idx]; }
-    const Item &operator [](const int idx) const { return items[idx]; }
```

Comment:
This unused constant index operator cannot be implemented using existing non-constant raw(). Implementing it is possible, but a proper implementation should change existing raw() callers. I think it is best to just remove this unused operator for now. FlexibleArray (and its users) should be polished further, but this build-fixing PR is not about such polishing.